### DISCUSSION TITLE
MeshAgent own code signing certificate

### DISCUSSION THREAD

**Discussion by HorizonGami** [amidaware/tacticalrmm#1496] [General]
Hello,

I am testing TacticalRMM in my lab.
The solution seems perfect and ideal.

One small question, I would like to sign the agent myself with a Code Signing certificate (Sectigo Code Signing).
Could you please tell me how to proceed ?

Can you tell me if I should also sign the TacticalRMM executable to avoid malware detection by the AV ?

Thank you in advance.

**Comment by silversword411** [comment]
As code signing is one of the main ways we're funding the project we do not support doing it yourself.
